docs/home/contribute/participating.md

```diff
@@ -0,0 +1,75 @@
+
+Our automation will reviewers to use for reviews on pull requests, and a contributor may request your review with an `/assign [your_github_handle]` comment in the pull request. To indicate your approval of a pull request, a reviewer should add an `/approve` comment to the pull request. A reviewer may also use add a `/hold` comment on the pull request to prevent the pull request from being merged. Another reviewer or approver may also remove that hold with the comment `/hold cancel`.
+
+When a reviewer is assigned a pull request to review it is not a sole responsibility, and any other reviewer may also offer their opinions on the pull request. If a reviewer is requested, it is generally expected that the PR will be left to that reviewer to do their editorial pass on the content. For small and/or trivial pull requests, the assigned reviewer may be skipped if an approver thinks the content is sufficient and well understood. 
+
+- Approvers
+
+Approvers are the individuals with final approval over a PR for merging. An approver can indicate `/lgtm` or `/approve` to have a pull request merged, and all pull requests require at least one approver to provide their vote for the PR to be merged. In general, this is done with an `/approve` comment, expecting an `/lgtm` to come from another reviewer. 
+
+A special case is where a pull request is a quick fix, typo, etc - in those cases, an approver may enter `lgtm`, which is regarded as a combination of both and accepted without further review.
+
+### Teams and groups within SIG-DOCS
+
+You can get an overview of [SIG-DOCS from the community github repo](https://github.com/kubernetes/community/tree/master/sig-docs). The SIG-DOCS group has two teams defined on github:
+ - [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
+ - [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
+
+These groups are used for access to the [website repository](https://github.com/kubernetes/website), which houses the content that hosted at this site. Both can be referenced with their `@name` in github comments to communicate with everyone in that group.
+
+These groups overlap, but aren't identical, to the automation tooling that Kubernetes uses as a project. For assignment of issues, pul requests, and to support PR approvals, the automation uses an OWNERS file for the repository. The approvers and reviewers for PR assignment and automation tooling work from the OWNERS file. 
+
+If you wish to volunteer to be a reviewer or an approver, make a pull request and add your github handle to the relevant section in the OWNERS file.
```

Comment:
Replace with:
```
To volunteer as a reviewer or an approver, make a pull request and add your GitHub handle to the relevant section in the OWNERS file.

**Note:** Reviewers and maintainers must meet requirements for participation. For more information, see the [Kubernetes community](https://github.com/kubernetes/community/blob/master/community-membership.md#member) repository.
{: .note}
```
So:
> **Note:** Reviewers and maintainers must meet requirements for participation. For more information, see the [Kubernetes community repository](https://github.com/kubernetes/community/blob/master/community-membership.md#member).

Reply:
Ooh - nice, missed the ‘requirements for participation’ link earlier, glad you suggested it!